modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-10-0_{context}"]
+= Release notes for {gitops-title} 1.10.0
+
+{gitops-title} 1.10.0 is now available on {product-title} 4.12, 4.13 and 4.14.
+
+[id="errata-updates-1-10.0_{context}"]
+== Errata updates
+
+[id="gitops-1-10-0-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX and RHEA-2023:YYYY - {gitops-title} 1.10.0 security update advisory
+
+Issued: 2023-09-27
```

Comment:
We might change the release date to 28th, we'll update soon

Reply:
Sure, keep me posted if need be.